developer-docs-site/docs/guides/running-a-local-multi-node-network.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Running a Local Mult-node Network"
+slug: "running-a-local-multi-node-network"
+---
+
+# Running a Local Mult-node Network
+
+This guide describes how to run a local network with multiple validator nodes and validator fullnodes. You will use the [Aptos Forge CLI](https://github.com/aptos-labs/aptos-core/tree/main/testsuite/forge-cli) for this.
```

Comment:
Should we maybe link to the actual README: https://github.com/aptos-labs/aptos-core/tree/main/testsuite/forge-cli/src?